checkov/cloudformation/cfn_utils.py

```diff
@@ -100,3 +105,42 @@ def get_folder_definitions(root_folder, excluded_paths: Optional[List[str]]):
 
     return definitions, definitions_raw
 
+
+def build_definitions_context(definitions, definitions_raw, root_folder):
+    definitions_context = {}
+    # iterate on the files
+    for file_path, file_path_definitions in definitions.items():
+        # iterate on the definitions (Parameters, Resources, Outputs...)
+        for file_path_definition, definition in file_path_definitions.items():
+            if isinstance(file_path_definition, str_node)\
+                    and file_path_definition.upper() in CF_DEFINITION_TYPES and isinstance(definition, dict_node):
+                # iterate on the actual objects of each definition
+                for attribute, attr_value in definition.items():
+                    if isinstance(attr_value, dict_node):
+                        start_line = attr_value.start_mark.line
+                        end_line = attr_value.end_mark.line
+                        code_lines = definitions_raw[file_path][start_line - 1: end_line]
+                        file_abs_path = create_file_abs_path(root_folder, file_path)
+                        dpath.new(definitions_context,
+                                  [file_abs_path, str(file_path_definition), str(attribute)],
+                                  {'start_line': start_line, 'end_line': end_line, 'code_lines': code_lines}
+                                  )
+                        if file_path_definition.upper() == "RESOURCES":
```

Comment:
`"RESOURCES"` is found here and on the new `frozenset` try re-using reserved attribute names